buildmaster/NMCPD/filter.py

```diff
@@ -0,0 +1,69 @@
+#Filter for NMCPD
+
+import sys
+import yaml
+
+def filter_NMCPD():
+    with open('metadata.yaml', 'r') as file:
+        metadata = yaml.safe_load(file)
+
+    version = metadata['hepdata']['version']
+    tables  = metadata['hepdata']['tables']
+
+    data_central = []
+    kin = []
+    error = []
+    error_nuc = []
+    
+    for i in tables:
+        hepdata_tables="rawdata/HEPData-ins426595-"+version+"-Table_"+str(i)+".yaml"
+        with open(hepdata_tables, 'r') as file:
+            input = yaml.safe_load(file)
```

Comment:
I will also avoid using `input` as a variable as that is a built-in function in python.
```suggestion
        input_yaml = download_hepdata_table(tables_dict, i)
```